indexer/src/main/java/au/org/aodn/esindexer/utils/MapperUtils.java

```diff
@@ -32,37 +35,23 @@ public static class Contacts {
     }
 
     public static List<String> mapContactsRole(CIResponsibilityType2 ciResponsibility) {
-        if (
-                ciResponsibility == null
-                || ciResponsibility.getRole() == null
-                || ciResponsibility.getRole().getCIRoleCode() == null
-                || ciResponsibility.getRole().getCIRoleCode().getCodeListValue() == null
-        ) {
-            return Collections.emptyList();
-        }
-        return Collections.singletonList(ciResponsibility.getRole().getCIRoleCode().getCodeListValue());
+        var contactsRole = getNullIfNullPointer(() -> ciResponsibility.getRole().getCIRoleCode().getCodeListValue());
```

Comment:
`getNullIfNullPointer` can this has a clearer name?

Reply:
Do you have recommendations? Yes I also considered a lot about this name yesterday. I wanted to have a name with shorter words and better express, but this one is currently the best one from my brain.

Reply:
maybe, as the whole reason to do this is safely handling null pointer exception
- nullSafeGet
- safeGetValue

least related 
- getValueOrNull

Reply:
Thank you! very helpful 👍 . I will pick one of your options and also leave the doc comment there (for someone may join our project in the future)